files/init-container.sh

```diff
@@ -23,6 +23,9 @@ case $GIT_BRANCH in
 	qemu-upstream)
 		XS_REPO_KEY=aHR0cDovL3hhcGktcmVwb3MuczMtd2Vic2l0ZS11cy1lYXN0LTEuYW1hem9uYXdzLmNvbS83ZWEzNzIxMi05MDc5LWUzMjEtNTdhYi0xZTQ5ZWFmYzBkY2YvZG9tYWluMA==
 		;;
+	vgpu-migration)
+		XS_REPO_KEY=aHR0cDovL3hhcGktcmVwb3MuczMtd2Vic2l0ZS11cy1lYXN0LTEuYW1hem9uYXdzLmNvbS9hNjIxMTk2MS04ZGFkLTQzYjctOGFlMy1iOTQ0YzIxNzkxNGEvZG9tYWluMA==
+		;;
```

Comment:
So this will work for the `vgpu-migration` branch of xenopsd by default, but not for the `emu-manager` branch, but I assume you're happy with this behaviour? You can always add a `GIT_BRANCH=vgpu-migration` to `emu-manager` to make it work.